nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -115,16 +132,19 @@ def create_nncf_graph(model: torch.fx.GraphModule) -> PTNNCFGraph:
         :param model: torch fx GraphModule.
         :return: NNCFGraph.
         """
-
         nncf_graph = PTNNCFGraph()
 
+        const_targets_counter = Counter([node.target for node in model.graph.nodes if node.op == "get_attr"])
         for source_node in model.graph.nodes:
             node_type, node_metatype = GraphConverter._get_node_type_and_metatype(source_node, model)
+            node_metatype = GraphConverter._map_fx_unique_metatypes(source_node, node_metatype)
+            is_shared_node = False
```

Comment:
Redundant line

Reply:
I'm not sure if I understood it right but if you are talking about the `node_metatype` assignment, this is because the metatype is initially mapped to the pytorch metatypes and then uses `_map_fx_unique_metatypes()` method to correctly map the metatype for special cases in torch fx(embedding for now).

Reply:
Oh, Alright. Done!